source/Calamari/Kubernetes/Scripts/KubectlBashContext.sh

```diff
@@ -110,83 +123,97 @@ function setup_context {
       K8S_Azure_Cluster+="-admin"
     fi
     kubectl config set-context $K8S_Azure_Cluster --namespace=$Octopus_K8S_Namespace
+  elif [[ $IsUsingPodServiceAccount ]]; then
+    kubectl config set-cluster octocluster --server=$Octopus_K8S_ClusterUrl
+    
+    if [[ -z $Octopus_K8s_Server_Cert_From_Path ]]; then
+      kubectl config set-cluster octocluster --insecure-skip-tls-verify=$Octopus_K8S_SkipTlsVerification
+    else
+      kubectl config set-cluster octocluster --certificate-authority=$Octopus_K8s_Server_Cert_Path
```

Comment:
If the cert path is not defined, this call can be skipped.

Reply:
I think the condition of the `if` statement has covered this case, hasn't it? it will be skipped if `$Octopus_K8s_Server_Cert_From_Path` is empty (due to the path `$Octopus_K8s_Server_Cert_Path` is invalid or empty)

Reply:
Ah, that is try. Please ignore this comment.